server/webhook.go

```diff
@@ -132,7 +132,7 @@ func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if errHandler != nil {
-		p.API.LogDebug("Error when handling webhook event", "err", err)
+		p.API.LogDebug("Error when handling webhook event", "err", errHandler)
```

Comment:
Why even use a different `error` variable?

Reply:
@hanzei `err` is reassigned in the block above this. I agree this could be implemented differently, but how exactly do you see this being implemented?

Reply:
Can't we use `err` here?

Reply:
@hanzei Should we use a different variable ame above this block then? Maybe `namespaceErr`?

Reply:
Can't we share the same variable?

Reply:
@hanzei I think we would need to change the order of operations in the code to do that right? I was thinking `isNamespaceAllowed` takes precedence over checking an error, based on the current code. I don't see how they could use the same var name without changing the order of code.

Reply:
Correct. Why does the `isNamespaceAllowed` check take precedence anyway? That seems odd.

Reply:
@hanzei I don't know, but I don't want to change the logic here. Just fixing the typo

Would we want to log an error when it occurs and the namespace is not allowed? I'm not sure